test/pipeline_integration_test.exs

```diff
@@ -0,0 +1,75 @@
+ExUnit.start()
+
+defmodule Membrane.VideoCompositor.PipelineTest do
```

Comment:
Both of these tests take a lot of time. Tag them as `:long` (and exclude `:long` tests in `test_helper`) and add 1 / 2 seconds long inputs